index.js

```diff
@@ -1,11 +1,33 @@
-function scuberGreetingForFeet(){
-  // Write your code here!
+function scuberGreetingForFeet(rideDistance){
+  let scuberGreeting
+  if(rideDistance <= 400){
+    scuberGreeting = 'This one is on me!'
+  }else if(rideDistance > 400 && rideDistance < 2000){
```

Comment:
You can exclude `rideDistance > 400 &&` since it's already covered by the first part of the conditional.
```suggestion
  }else if(rideDistance < 2000){
```